### ISSUE TITLE
multiple convolution kernels bug

### ISSUE THREAD

**Issue by tokenwood** [BradLarson/GPUImage#646]
Hi !

I am trying to apply two different convolution kernels to one image.

so I initialize my kernels : 

GPUImage3x3ConvolutionFilter *filter1 = [[GPUImage3x3ConvolutionFilter alloc] init];
GPUImage3x3ConvolutionFilter *filter2 = [[GPUImage3x3ConvolutionFilter alloc] init];

I set my kernels : 

[filter1 setConvolutionKernel:kernel1];
[filter2 setConvolutionKernel:kernel2];

and now I apply my two convolution filters to my image : 

[image addTarget : filter1]
[image addTarget : filter2]
[image processImage]

but when I look at [filter1 imageFromCurrentlyProcessedOutput], it show an image filtered by kernel2 insetad of kernel1 !

in other words, the last kernel you set is the one that will be applied to all convolutionfilters.
Am I doing something wrong? is there an other way?

Thanks!
